software/chibisafe.yml

```
@@ -0,0 +1,13 @@
+name: "Chibisafe"
+website_url: "https://chibisafe.moe"
+source_code_url: "https://github.com/chibisafe/chibisafe"
+description: "Blazing fast file uploader and awesome bunker written in node!"
+licenses:
+  - MIT
+platforms:
+  - JavaScript
+  - Nodejs
+  - Docker
```

JS isnt needed as Nodejs is already present / changed order to match docs
```suggestion
platforms:
  - Docker
  - Nodejs
```